test-projects/lambda-client/amplify/functions/todo-count/handler.ts

```diff
@@ -1,14 +1,17 @@
 import type { Handler } from "aws-lambda";
-import { Amplify } from "aws-amplify";
-import { generateClient } from "aws-amplify/data";
-import type { Schema } from "../../data/resource";
-import { resourceConfig, libraryOptions } from "$amplify/client-config/todo-count"
+// import { Amplify } from "aws-amplify";
+// import { generateClient } from "aws-amplify/data";
+// import type { Schema } from "../../data/resource";
+// import { resourceConfig, libraryOptions } from "$amplify/client-config/todo-count"
 
-Amplify.configure(resourceConfig, libraryOptions);
-
-const client = generateClient<Schema>();
+// Amplify.configure(resourceConfig, libraryOptions);
+//
+// const client = generateClient<Schema>();
 
 export const handler: Handler = async () => {
-  const todos = (await client.models.Todo.list()).data;
-  return todos.length;
+
+  console.log(JSON.stringify(process.env, null, 2));
```

Comment:
Added this to see if plumbing works.

![image](https://github.com/user-attachments/assets/a81461d1-8ab0-4bb0-b173-9733d5a3c8e1)